python/ray/autoscaler/_private/command_runner.py

```diff
@@ -584,6 +584,9 @@ def __init__(self, docker_config, **common_args):
         self.docker_config = docker_config
         self.home_dir = None
         self.initialized = False
+        # Optionally use 'podman' instead of 'docker'
+        use_podman = docker_config.get("use_podman", False)
```

Comment:
This looks good!, Can you make sure to add this to:
[ray-sechema.json](https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/ray-schema.json).

Reply:
Sure thing!

Reply:
Looks good!